bluemira/base/reactor.py

```diff
@@ -94,6 +275,12 @@ def component(
         """Return the component tree."""
         return self._build_component_tree(with_components)
 
+    def build_time(self) -> float:
+        """
+        Get time since initialisation
+        """
+        return time.perf_counter() - self.start_time
```

Comment:
This method may be a bit misleading if the `Reactor` is being used in an interactive environment, e.g., in a Jupyter Notebook. The docstring is clear that it's time since initialization, not time spent building, but the function name is less clear on that. `time_since_init` is more explicit, but naming this is a bit tricky, I'll admit. Happy to leave it if you are.

Reply:
yep prefer your name, changed!